docs/faq/proof-of-stake/general.md

```diff
@@ -132,59 +115,56 @@ Currently, a platform for the community to submit and advocate a voting agenda i
 
 #### 10. What is hardfork voting? 
 
-Like any other cryptocurrency Decred might need to hardfork at some point.
+A more up-to-date version of this information, with a diagram, is available on [adgenda voting](/getting-started/user-guides/agenda-voting/) page.
```

Comment:
This page has moved. And also internal links should be relative and point to `.md` files to take advantage of mkdocs validation.

```
../../governance/consensus-rules-voting.md
```

Reply:
Have gone with suggested link, as I think that's what we want. 

> A more up-to-date version of this information, with a diagram, is available on [consensus rules voting](../../governance/consensus-rules-voting.md) page.